lib/src/simple_op_store.rs

```diff
@@ -100,7 +101,9 @@ impl OpStore for SimpleOpStore {
 
         temp_file.as_file().write_all(&proto_bytes)?;
 
-        let id = ViewId::new(Blake2b512::digest(&proto_bytes).to_vec());
+        let mut hasher = Blake2b512::default();
+        view.hash(&mut hasher);
+        let id = ViewId::new(hasher.finalize().to_vec());
```

Comment:
Do you want to also demo the functionality on the native commit backend (`local_backend.rs`)?

Reply:
Sure!